packages/create-cedar-rsc-app/eslint.config.js

```diff
@@ -36,6 +36,26 @@ export default tseslint.config(
     },
   },
   perfectionist.configs['recommended-natural'],
+  {
+    // Apply the same rules used in .eslintrc.js
```

Comment:
This linter settings fixes the pre-push issue:
<img width="1394" height="694" alt="image" src="https://github.com/user-attachments/assets/96112b63-46b7-4491-a673-1656bb176719" />